scripts/build/debian/config.json

```diff
@@ -4,7 +4,7 @@
   "genericName": "Etcher",
   "homepage": "https://etcher.io/",
   "icon": "assets/icon.png",
-  "bin": "etcher-electron.sh",
+  "bin": "etcher",
```

Comment:
Good point. I added a workaround to rename the binary to `etcher-electron` and reference it as such. It will be fixed the "right" way on the `electron-builder` PR.